src/plugin/systems.rs

```diff
@@ -367,7 +367,10 @@ pub fn apply_rigid_body_user_changes(
 /// System responsible for applying changes the user made to a joint component.
 pub fn apply_joint_user_changes(
     mut context: ResMut<RapierContext>,
-    changed_impulse_joints: Query<(&RapierImpulseJointHandle, &ImpulseJoint), Changed<RigidBody>>,
+    changed_impulse_joints: Query<
+        (&RapierImpulseJointHandle, &ImpulseJoint),
+        Or<(Changed<RigidBody>, Changed<ImpulseJoint>)>,
+    >,
     changed_multibody_joints: Query<
         (&RapierMultibodyJointHandle, &MultibodyJoint),
         Changed<GlobalTransform>,
```

Comment:
The previous `Changed` condition was incorrect, so no need for an `Or` here:

```suggestion
    changed_impulse_joints: Query<
        (&RapierImpulseJointHandle, &ImpulseJoint),
       Changed<ImpulseJoint>,
    >,
    changed_multibody_joints: Query<
        (&RapierMultibodyJointHandle, &MultibodyJoint),
        Changed<MultibodyJoint>,
```

Reply:
should be good now